client/app/bundles/course/users/components/UserProfileSkills.tsx

```diff
@@ -17,7 +17,7 @@ interface Props extends WrappedComponentProps {
 const translations = defineMessages({
   topicMasteryHeader: {
     id: 'course.users.show.topicMasteryHeader',
-    defaultMessage: 'Topic Mastery',
+    defaultMessage: 'Skill Mastery',
```

Comment:
Any reason for this change?

Reply:
@cysjonathan mentioned that this should be named Skill instead since it is also part of the Skills and Skill Branches.